go/analysis/passes/printf/printf.go

```diff
@@ -66,6 +68,51 @@ argument list. Otherwise it is assumed to be Print-like, taking a list
 of arguments with no format string.
 `
 
+type PrintKind int
```

Comment:
Call this type Kind. It needs a doc comment.

Reply:
I added a comment. Let me know what you think.

Sometimes we refer to these functions as "Print wrappers" and sometimes as "behaving like Print". Do you have a preference here? "Behaving like Print" is less clear, but I think more accurate because `Result.Kind()` is going to return `KindPrint` for `fmt.Print` itself, not just its wrappers.